src/pages/changePassword.tsx

```diff
@@ -0,0 +1,140 @@
+                                            errorClass="govuk-input--error"
+                                        >
+                                            <input
+                                                className="govuk-input"
+                                                id="new-password"
+                                                name="newPassword"
+                                                type="password"
+                                                aria-describedby="change-password-page-hint"
+                                                spellCheck="false"
+                                            />
+                                        </FormElementWrapper>
+                                    </div>
+
+                                    <div className="govuk-form-group">
+                                        <label className="govuk-label" htmlFor="confirm-new-password">
+                                            Confirm new password
+                                        </label>
+                                        <FormElementWrapper
+                                            errors={errors}
+                                            errorId="confirm-new-password"
+                                            errorClass="govuk-input--error"
+                                        >
+                                            <input
+                                                className="govuk-input"
+                                                id="confirm-new-password"
+                                                name="confirmNewPassword"
+                                                type="password"
+                                                aria-describedby="change-password-page-hint"
+                                                spellCheck="false"
+                                            />
+                                        </FormElementWrapper>
+                                    </div>
+                                </div>
+                            </div>
+                            <input
+                                type="submit"
+                                name="changePassword"
+                                value="Change Password"
+                                id="change-password-button"
+                                data-module="govuk-button"
+                                className="govuk-button govuk-!-margin-right-1"
+                            />
+                            <a
+                                href="/account"
+                                role="button"
+                                draggable="false"
+                                className="govuk-button govuk-button--secondary"
+                                data-module="govuk-button"
+                                id="cancel-change-password-button"
+                            >
+                                Cancel
```

Comment:
Just seen we have a Cancel button on the mock, which is presumably why we have this one here. Feels pretty out of whack with how we currently let the user submit information in a form; i.e. this is the only cancel button we have. Do we really need it?

Reply:
Yeah it is a bit odd we don't have it elsewhere. It does help this page to be a bit more user friendly though. For example, if a user accidentally clicks the 'Change' button on the account page, there's no obvious way on the page for the user to get back. It could be we need a Cancel button elsewhere rather than removing this one? Another good thing to test in a user research session 👍